### PR TITLE
[Tests] Add cypress e2e test for deleting added city

### PR DESCRIPTION
**Pull request by dpakach**
Add cypress End-To-End tests for:

1. Delete a city from the weather table.

2. Add multiple cities and delete them one by one.

### CODE REVIEW

File: cypress/integration/deleteCity.spec.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    })
+
+    describe('User can delete a city from the list', () => {
+        it('should be possible to remove a city after adding one', () => {
+            cy.searchForCity(tokyo)
+            cy.get('.city-name-cell').should('be.visible')
+            cy.waitForAnimationToFinish()
+            cy.get('.city-name-cell')
+                .contains(tokyo.name)
+                .click()
```
Comment (Talank) [inline cypress/integration/deleteCity.spec.js:19]: can you make a custom command for performing this task!

**Review by Talank — APPROVED** [review]
LGTM :+1:

File: cypress/fixtures/cityCountry.json [hunk 2]
```diff
@@ -1,10 +1,14 @@
 {
-  "1": {
-    "city": "Tokyo",
+  "tokyo": {
+    "name": "Tokyo",
     "country": "Japan"
   },
-  "2": {
```
Comment (bartius-nigel) [inline cypress/fixtures/cityCountry.json:6]: awesome.

File: cypress/integration/searchCity.spec.js [hunk 3]
```diff
@@ -9,23 +11,23 @@ context('Weather', () => {
 
     describe('User can search for weather in a city', () => {
         it('should be possible to search the weather of a single city at a time', () => {
-            cy.searchForCity(cityCountry['1'].city,cityCountry['1'].country)
-            cy.get('.city-name-cell').should('be.visible')
+            cy.searchForCity(tokyo)
+            cy.get('.name-name-cell').should('be.visible')
```
Comment (bartius-nigel) [inline cypress/integration/searchCity.spec.js:15]: id prefer this to stay as '.city-name-cell' if thats possible.

File: cypress/support/commands.js [hunk 4]
```diff
@@ -2,8 +2,8 @@ Cypress.Commands.add('waitForAnimationToFinish', () => {
     cy.wait(3000)
 })
 
-Cypress.Commands.add('searchForCity', (city, country) => {
```
Comment (bartius-nigel) [inline cypress/support/commands.js:5]: Same here, id prefer this to stay as 'city'

**Review by bartius-nigel — CHANGES_REQUESTED** [review]
This looks great, thank you for catching some great issues!

**Comment by dpakach** [comment]
@bartius-nigel, sorry was busy yesterday, but I've fixed all the issues. Please have a look

**Review by bartius-dev — APPROVED** [review]

**Pull request merged**